src/tests/TestHelper.ts

```diff
@@ -128,6 +128,7 @@ export module TestHelper {
             program = ts.createProgram([inputFileOrScript], { });
         }
         if (inputFileOrScript.match(/.*\.ts(x)?$/)) {
+            /* tslint:disable:non-literal-fs-path */
```

Comment:
Ah, `tslint:disable` disables the rule for the rest of the file. You're looking for:

```typescript
// tslint:disable-next-line non-literal-fs-path
```

Reply:
Oops, dind't know about that, should be fixed now.